lib/src/modules/task.cc

```diff
@@ -453,7 +549,8 @@ static Implementation selectImplementation(std::vector<lth_jc::JsonContainer> co
 
     if (impl != implementations.cend()) {
         auto input_method = impl->getWithDefault<std::string>("input_method", {});
-        return Implementation { impl->get<std::string>("name"), input_method };
+        auto files = impl->getWithDefault<std::string>("files", {});
```

Comment:
This should be getting a vector of strings, i.e. `impl->getWithDefault<std::vector<std::string>>("files", {});`.